flink-streaming-connectors/flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/container/RedisContainer.java

```diff
@@ -68,12 +68,12 @@ public RedisContainer(final JedisSentinelPool sentinelPool) {
 	 */
 	@Override
 	public void close() throws IOException {
-		if (this.jedisPool != null) {
-			this.jedisPool.close();
-		}
-		if (this.jedisSentinelPool != null) {
-			this.jedisSentinelPool.close();
-		}
+		getInstance().close();
+	}
+
+	@Override
+	public void open() throws Exception {
+		getInstance().echo("Test");
```

Comment:
Is this the only way to test a connection? If so, then it should be documented properly that this is used to test the connection.